fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend

```diff
@@ -224,19 +229,31 @@ __all__ = ["«cls.name»", "main"]
 
 def enumClasses(PogoDeviceClass cls) '''
 «IF cls.attributes!==null»
-«FOR attr:cls.attributes»
-«IF attr.dataType.pythonTypeHL.equalsIgnoreCase("'DevEnum'")»
-
-
-class «attr.name.toFirstUpper»(IntEnum):
-    """«attr.name.toFirstUpper» enum."""
-    «IF attr.enumLabels!==null»
-    «IF attr.enumLabels.size>0»
-    «attr.enumLabelsWithNumber»
-    «ENDIF»
-    «ENDIF»
-«ENDIF»
-«ENDFOR»
+	«FOR attr:cls.attributes»
+	«IF attr.dataType.pythonTypeHL.equalsIgnoreCase("'DevEnum'")»
+		«IF attr.checkEnumLabels == "valid"»
+		
+		
+		class «attr.name.toFirstUpper»(IntEnum):
+		    """Python enumerated type for «attr.name.toFirstUpper» attribute."""
+		    «IF attr.enumLabels!==null»
+		    «IF attr.enumLabels.size>0»
+		    «attr.enumLabelsWithNumber»
+    		«ENDIF»
+    		«ENDIF»
+		«ENDIF»
+		«IF attr.checkEnumLabels == "invalid"»
+		
+		
+		«attr.name.toFirstUpper» = enum.IntEnum(
+		    value='«attr.name.toFirstUpper»',

```

Comment:
Rather `"` instead of `'` for this string, to match the double quotes used for the labels.

Reply:
Okay. Done.